libs/wallet/src/web3-react/hooks/useSafeAppsSdkInfo.ts

```diff
@@ -1,22 +1,24 @@
-import { useEffect, useState } from 'react'
-
 import type { SafeInfo } from '@safe-global/safe-apps-sdk'
 
-import { useSafeAppsSdk } from './useSafeAppsSdk'
+import { useAtom } from 'jotai/index'
+import { gnosisSafeInfoAtom } from '@cowprotocol/wallet'
 
 export type GnosisSafeSdkInfo = SafeInfo
 
 export function useSafeAppsSdkInfo(): GnosisSafeSdkInfo | null {
-  const [gnosisSafeInfo, setGnosisSafeInfo] = useState<GnosisSafeSdkInfo | null>(null)
-  const safeAppsSdk = useSafeAppsSdk()
+  const gnosisSafeInfo = useAtom(gnosisSafeInfoAtom)[0]
```

Comment:
This hooks was exposing the SDK information extracted form the connector. 
Now its exposing the global state.

I think this helped with the re-renders, but IMO is not correct. 

A first small issue is that being called SafeAppsSdkInfo should return the info coming from the SDK, but most of its info comes from the API. This is conceptually misleading. 

What is right, is that we should expose this `gnosisSafeInfo` using a hook, AND whomever is using `useSafeAppsSdkInfo` should use that one instead. 

Some other small thing, is, the implementation to export the global info should be simpler:
something like:
```
export function useSafeInfo() {
    return useAtomValue(safeInfoAtom)
}
```

then hooks like `useIsSafeWallet()` should use `useSafeInfo()` and not `useSafeAppsSdkInfo()`